web-frontend/src/utils/const.ts

```diff
@@ -1,15 +1,13 @@
-// TODO: add network store to be able to dynamically fetch correct contract
-
-export enum MnsContract {
+export enum networks {
   mainnet = 'mainnet',
   buildnet = 'buildnet',
 }
 
 export const contracts = {
-  [MnsContract.mainnet]: {
-    MNScontract: '',
+  [networks.mainnet]: {
```

Comment:
maybe use the chain id as key

Reply:
the id's are bigInt's, i would need to convert them as " computed property name must be of type 'string', 'number', 'symbol', or 'any'." should i proceed ?